workflow-core/src/commonMain/kotlin/com/squareup/workflow1/WorkerWorkflow.kt

```diff
@@ -89,7 +89,8 @@ private class EmitWorkerOutputAction<P, S, O>(
   private val output: O
 ) : WorkflowAction<P, S, O>() {
   override fun toString(): String =
-    "${EmitWorkerOutputAction::class.qualifiedName}(worker=$worker, key=\"$renderKey\")"
+    "${WorkflowIdentifierTypeHelper.uniqueName(EmitWorkerOutputAction::class)}" +
```

Comment:
can you add another commit to remove the redundant string template? The Ide will prompt you.